docs: document Code node

aarthificial opened the pull request.

hhenrichsen commented on packages/docs/docs/components/code/index.mdx:
````diff
@@ -0,0 +1,791 @@
+});
+```
+
+## Multiple Languages
````
This (or a section after this) might be a good place to document how to use a Lezer dialect, too:
```tsx
LezerHighlighter.registerParser(
  javascriptParser.configure({ dialect: "jsx" }),
  "jsx"
);
LezerHighlighter.registerParser(
  javascriptParser.configure({ dialect: "ts" }),
  "ts"
);
```

aarthificial replied:
True, I'll add this in the next PR to not bloat up this one.

hhenrichsen reviewed (approved):
Mostly a bunch of grammar suggestions based on Google's style guide, but a couple questions mixed in.

aarthificial commented on packages/docs/docs/components/code-block.mdx:
```diff
@@ -1,12 +1,19 @@
 slug: /code-block
 ---
 
-# CodeBlock
```
True, I'll do that in a separate PR because I have some more changes related to the node being experimental to do

aarthificial replied:
Changed this in #1000

aarthificial commented on packages/docs/docs/components/code/index.mdx:
```diff
@@ -0,0 +1,792 @@
+to fix this, we recommend you use Firefox for rendering.
+
+[tag-function]:
+  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates
```
Agree, also:
- using custom draw hooks
- creating/using custom themes

aarthificial commented:
@hhenrichsen Thanks for the review! I changed all instances of `we` to `you` as suggested. However, I changed the verb in some places to make the tone less authoritarian(?).
For example: "you'll use" -> "you can use" / "you should use"

hhenrichsen commented on packages/docs/docs/components/code/index.mdx:
```diff
@@ -0,0 +1,790 @@
+  or tabs at the beginning of each line would be included in the snippet.
+
+- The backslash character (`\`) at the very beginning is used to escape the
+  first newline character. This lets us start the snippet on a new line without
```
nit: us -> you

aarthificial replied:
Fixed in #1002

hhenrichsen commented on packages/docs/docs/components/code/index.mdx:
````diff
@@ -0,0 +1,790 @@
+  ```
+
+Template strings allow you to easily include variables in your code snippets
+with `${}` syntax. In the example below, `${name}` is replaced with the value of
````
I think I prefer "with the `${}` syntax" here.

aarthificial replied:
Fixed in #1002

hhenrichsen reviewed (approved):
A couple other things I noticed going through again, but not any egregious blockers.
